### PR TITLE
Fix help string for "tctl version"

### PR DESCRIPTION
**Pull request by justinas**
`tctl version`, just as `tsh version`, prints out the version of the binary itself, not the version of the cluster it's connected to, e.g.:

```console
$ tctl version
Teleport v9.1.2 git: go1.17.10
```

The cluster version can be discovered by running `tctl status`:

```console
$ tctl status
Cluster  localhost
Version  9.1.2
Host CA  never updated
User CA  never updated
Jwt CA   never updated
CA pin   sha256:5ab93c1a05d0fe58c66c8f3ad2e406d15e45446b606919c2e03bb1d3a8ac68e6
```

### CODE REVIEW

**Review by codingllama — APPROVED** [review]
Thanks!

File: docs/pages/setup/reference/cli.mdx [hunk 1]
```diff
@@ -1431,7 +1431,7 @@ $ tctl top http://127.0.0.1:3000 5s
 
 ### tctl version
 
-Print cluster version:
+Print client version:
```
Comment (ptgott) [inline docs/pages/setup/reference/cli.mdx:1434]:
```suggestion
Print the version of your `tctl` binary:
```
To make this slightly more precise

File: tool/tctl/common/tctl.go [hunk 2]
```diff
@@ -133,7 +133,7 @@ func Run(commands []CLICommand) {
 		BoolVar(&ccf.Insecure)
 
 	// "version" command is always available:
-	ver := app.Command("version", "Print cluster version")
+	ver := app.Command("version", "Print the version")
```
Comment (ptgott) [inline tool/tctl/common/tctl.go:136]:
```suggestion
	ver := app.Command("version", "Print the version of your tctl binary")
```

**Review by EdwardDowling — APPROVED** [review]

File: docs/pages/setup/reference/cli.mdx (hunk 1 above)
Comment (justinas) [reply docs/pages/setup/reference/cli.mdx]: Sorry for the holdup! Updated this and tsh's equivalents for consistency.

File: tool/tsh/tsh.go [hunk 3]
```diff
@@ -495,7 +495,7 @@ func Run(ctx context.Context, args []string, opts ...cliOption) error {
 		EnumVar(&cf.MFAMode, modes...)
 	app.HelpFlag.Short('h')
 
-	ver := app.Command("version", "Print the version")
+	ver := app.Command("version", "Print the version of your tsh binary")
```
Comment (codingllama) [inline tool/tsh/tsh.go:498]: Maybe change `teleport`'s help too?

**Review by codingllama — APPROVED** [review]

**Review by ptgott — APPROVED** [review]

**Pull request merged**